sonic_installer/main.py

```diff
@@ -306,6 +306,15 @@ def update_sonic_environment(bootloader, binary_image_version):
             umount(new_image_mount)
 
 
+def get_docker_opts():
+    """ Get options dockerd is started with """
+    with open("/var/run/docker.pid") as pid_file:
```

Comment:
Check if we can know the pid file path from env ?

Reply:
https://serverfault.com/questions/417892/how-to-find-the-pid-file-for-a-given-process